src/Microsoft.ML.Transforms/NormalizerCatalog.cs

```diff
@@ -159,7 +159,53 @@ public static class NormalizationCatalog
             bool useCdf = NormalizingEstimator.Defaults.LogMeanVarCdf) =>
             new NormalizingEstimator(CatalogUtils.GetEnvironment(catalog),
                 columns.Select(column =>
-                    new NormalizingEstimator.LogMeanVarianceColumnOptions(column.OutputColumnName, column.InputColumnName, maximumExampleCount, useCdf)).ToArray());
+                    new NormalizingEstimator.LogMeanVarianceColumnOptions(column.OutputColumnName, column.InputColumnName, maximumExampleCount, useCdf, false)).ToArray());
+
+        /// <summary>
+        /// Create a <see cref="NormalizingEstimator"/>, which normalizes based on the computed mean and variance of the logarithm of the data.
+        /// </summary>
+        /// <param name="catalog">The transform catalog</param>
+        /// <param name="fixZero">TODO</param>
+        /// <param name="outputColumnName">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.
+        ///                                The data type on this column is the same as the input column.</param>
+        /// <param name="inputColumnName">Name of the column to transform. If set to <see langword="null"/>, the value of the <paramref name="outputColumnName"/> will be used as source.
+        ///                               The data type on this column should be <see cref="System.Single"/>, <see cref="System.Double"/> or a known-sized vector of those types.</param>
+        /// <param name="maximumExampleCount">Maximum number of examples used to train the normalizer.</param>
+        /// <param name="useCdf">Whether to use CDF as the output.</param>
+        /// <example>
+        /// <format type="text/markdown">
```

Comment:
Uses would love equations in documentations.

Reply:
I will add the equations here as well.

---
In reply to: [298260450](https://github.com/dotnet/machinelearning/pull/3916#discussion_r298260450) [](ancestors = 298260450)

Reply:
I am a little concerned about the equations. Yesterday I think I saw an issue. I will add those in a separate PR and check this in for now.

---
In reply to: [298310313](https://github.com/dotnet/machinelearning/pull/3916#discussion_r298310313) [](ancestors = 298310313,298260450)